### ISSUE TITLE
Combine RStudio Workbench with a second kubernetes cluster only to run R sessions

### ISSUE THREAD

**Issue by alessap** [rstudio/helm#144]
Is it possible to combine a RStudio Workbench running on a kubernetes (cluster01) with a second kubernetes cluster where only R sessions are run (cluster02). Moreover is it possible to couple a single RStudio Workbench instance with multiple kubernetes clusters where R sessions are run? If yes, is this achieavable by the current helm charts?

**Comment by colearendt** [comment]
Yes, it should be! However, I think it will be pretty tricky with the current helm charts. I would love to hear a bit more about why you are targeting this architecture, if possible!?

The basic pattern I anticipate off-hand would be:
- define other "clusters" in `launcher.conf` (other names `cluster01`, `cluster02`, etc. but all using the Kubernetes launcher)
- then in each cluster's `launcher.kubernetes.conf` (each would need a separate), you would need to set up the CA / credentials / etc. for each cluster (the cluster you are already in would get set up easily by the helm chart)
- users would then select between clusters using the "Cluster" drop down in the UI. By default we populate it with "Kubernetes," but that could just as easily be "cluster01", "cluster02," etc.

Please let us know if you have any further questions about that approach!
